examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -23,16 +22,18 @@ document.title = containerId;
 const dataObjectId = "dice";
 
 async function start(): Promise<void> {
-    const service = new TinyliciousService();
+    const containerConfig: FluidCreateContainerConfig = {
```

Comment:
Typescript didn't like the typing around `initialDataObjects` so I had to bring in the `FluidCreateContainerConfig` type here to make it happy. I would ideally not do this.